front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -281,34 +284,55 @@ const SimulationReportSheet = ({
                 const isFirstStep = index === 0;
                 const isLastStep = index === operationalPointsList.length - 1;
                 const prevStep = operationalPointsList[index - 1];
+                const isViaInSimulationPath = stdcmData.simulationPathSteps
+                  .slice(1, -1)
+                  .some((pathStep) => pathStep.name === step.name && pathStep.ch === step.ch);
+                const isViaWithoutStop = isViaInSimulationPath && step.duration === 0;
+                const isNotExtremity = !isFirstStep && !isLastStep;
+                const isStepWithDuration = step.duration !== 0 && !isLastStep;
                 return (
                   <TR
                     key={index}
-                    style={
-                      step.duration !== 0 && !isLastStep
-                        ? styles.simulation.blueRow
-                        : styles.simulation.tbody
-                    }
+                    style={isStepWithDuration ? styles.simulation.blueRow : styles.simulation.tbody}
                   >
-                    <TD style={styles.simulation.indexColumn}>{index + 1}</TD>
+                    <TD
+                      style={
+                        isViaWithoutStop
+                          ? styles.simulation.indexColumnPassageStop
+                          : styles.simulation.indexColumn
+                      }
+                    >
+                      {index + 1}
+                    </TD>
                     <View style={styles.simulation.opWidth}>
                       <TD
                         style={
-                          !isFirstStep && !isLastStep && step.duration !== 0
-                            ? styles.simulation.opStop
-                            : styles.simulation.td
+                          // eslint-disable-next-line no-nested-ternary
+                          isViaWithoutStop
+                            ? styles.simulation.opColumnPassageStop
+                            : isNotExtremity && step.duration !== 0
+                              ? styles.simulation.opStop
+                              : styles.simulation.td
                         }
                       >
-                        {!isFirstStep &&
-                        !isLastStep &&
+                        {isNotExtremity &&
+                        !isViaInSimulationPath &&
                         step.name === prevStep.name &&
                         step.duration === 0
```

Comment:
I think you can remove this condition.
=" will be displayed if the step is not a via or an extremity and if its name the prevStep.name are the same

```suggestion
```